scripts/OverlaySurvey.py

```diff
@@ -285,6 +323,18 @@ def main():
                                 help="output file for the augmented graph")
     parser_augment.set_defaults(func=augment)
 
+    parser_simplify = subparsers.add_parser('simplify',
+                                            help="simplify the master graph "
+                                            "and output it as json file")
+    parser_simplify.add_argument("-gmli",
```

Comment:
Can we clarify the usage here? Specifically, the input graph here should be augmented? It looks like you should also specify `required=True`

Reply:
Updated the comment, and added `required=True`!